README.md

```diff
@@ -90,6 +90,17 @@ helm install osiris/osiris-edge \
   --devel
 ```
 
+### Configure Osiris
+
+Osiris global configuration is minimal - because most of it will be done by the users
+with annotations on the Kubernetes resources.
+
+The following table lists the configurable parameters of the Helm chart and their default values.
+
+Parameter | Description | Default 
+--------- | ----------- | -------
+`zeroscaler.metricsCheckInterval` | The interval in which the zeroScaler would repeatedly track the pod http request metrics. The value is the number of seconds of the interval. | `150`
+
```

Comment:
Truly a nit, but I would somewhat prefer we include the "outer pipes" on each row of the table. I say this only because I've (anecdotally) observed that syntax to be more prevalent (especially among Helm charts) and assuming future contributors will amend this table over time, I want to favor the more familiar syntax.